libraries/Microsoft.Bot.Builder.Dialogs.Declarative/Converters/InterfaceConverter.cs

```diff
@@ -123,6 +131,7 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                         if (passTwo && refDialogName != null)
                         {
                             cachedRefDialogs[refDialogName] = result;
+                            this.resourceExplorer.UpdateResourceTokenCache(refDialogName, tokenToBuild, range);
```

Comment:
With the local cache, do we still need the resource token cache in resource explorer? I do see that there are many cases where there could be a cache miss in the interface converter but a cache hit in the resource explorer (i.e. multiple files pointing to the same dialogId). I'm curious what's the difference in experimental results for our current benchmark.

Reply:
When running the unit test for cyclic resources it was significantly faster to not have the cache in place, but when running the customer bot not having the cache in place raised the time from connecting to to newly started bot to the time it was ready to respond from 1:34 (min/sec) on my system to 2:04 (min/sec) average after multiple runs. So in the real world scenario the cache does seem to help in getting the bot loaded more quickly with little increase in overall memory consumption. With this in mind I'd recommend we keep the cache.